modules/calibration/data/dev_kit_advanced_ne-b/localization_conf/localization.conf

```diff
@@ -0,0 +1,213 @@
+--flagfile=/apollo/modules/common/data/global_flagfile.txt
+--noenable_gps_imu_interprolate
+
+# Redefine the map_dir of global_flagfile.txt
+#--map_dir=/apollo/modules/localization/testdata
+
+# The directory name of localization map.
+# type: string
+# default: local_map
+--local_map_name=local_map
+
+# Enable lidar-based localization.
+# type: bool
+# defalult: true
+--enable_lidar_localization=false
+
+# Localization mode, 0 for intensity, 1 for altitude, 2 for fusion.
+# type: int
+# default: 2
+--lidar_localization_mode=2
+
+# image yaw align mode, 0 for align off, 1 for fusion, 2 for fusion with multithread.
+# type: int
+# default: 2
+--lidar_yaw_align_mode=2
+
+# Lidar filter size
+# type: int
+# default: 11
+--lidar_filter_size=17
+
+# point cloud step
+# type: int
+# default: 2
+--point_cloud_step=2
+
+# The height from the center of lidar to ground plane
+# type: double
+# default: 1.80
+--lidar_height_default=1.80
```

Comment:
the value of lidar_height_default is different to the value of velodyne16_height.yaml